src/clj/fluree/db/json_ld/vocab.cljc

```diff
@@ -136,8 +191,9 @@
       (= const/$rdfs:subClassOf pid)
       (update pred-map sid with-subclass obj)
 
-      (= const/$owl:equivalentProperty pid)
-      (update-equivalent-properties db pred-map sid obj)
+      (or (= const/$owl:equivalentProperty pid)
```

Comment:
Because there are just two things being checked for here:
```
(or (= const/$owl:equivalentProperty pid)
    (= const/$rdfs:subPropertyOf pid))
```
and they are self-describing I'd prefer having them inline as opposed to consulting another function to see what they are. Also this is consistent with other related owl/rdfs checks around it, e.g. `const/$rdfs:subClassOf`.

I'm good changing it if you have a strong preference towards a separate function, but I think I'd understand better what is happening here by having it all together.

Reply:
yeah, that makes sense. agreed.